src/creature.h

```diff
@@ -300,7 +300,7 @@ class Creature
         /** Returns the intensity of the matching effect. Returns 0 if effect doesn't exist. */
         int get_effect_int(efftype_id eff_id, body_part bp = num_bp) const;
         /** Returns true if the creature resists an effect */
-        bool resists_effect(effect e);
+        bool resists_effect(effect e) const;
```

Comment:
Right, git this mixed up with `bound_mod_to_vals`. `bound_mod_to_vals` should be static.